docx_editor/exceptions.py

```
@@ -67,10 +67,96 @@ class CommentError(DocxEditError):
+        search_text: The text that was being searched for.
+        paragraph_ref: Paragraph reference the search was scoped to, or None
+            if the search was document-wide.
+        paragraph_preview: Current visible text of the scoped paragraph
+            (truncated to 80 chars with "..." suffix), or None if unscoped.
+        occurrence: 1-indexed occurrence number requested (for nth-match
+            lookups), or None otherwise.
+        total_occurrences: Actual number of occurrences found, or None for
+            non-occurrence lookups.
+    """
+
+    def __init__(
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)